clang/examples/PrintFunctionNames/PrintFunctionNames.cpp

```diff
@@ -72,7 +72,7 @@ class PrintFunctionsConsumer : public ASTConsumer {
           *sema.LateParsedTemplateMap.find(FD)->second;
       sema.LateTemplateParser(sema.OpaqueParser, LPT);
       llvm::errs() << "late-parsed-decl: \"" << FD->getNameAsString() << "\"\n";
-    }   
+    }
```

Comment:
This seems like an unrelated change.

Reply:
It's intentional. I need to make a change under `clang/` to test that CI doesn't compile and test Flang on Clang changes. I'll remove it after Windows CI passes.